ChatQuickstart/QuickstartChatManager.swift

```diff
@@ -0,0 +1,141 @@
+            // Join (or create) the general channel
+            let defaultChannel = "general"
+            if let channelsList = client.channelsList() {
+                channelsList.channel(withSidOrUniqueName: defaultChannel, completion: { (result, channel) in
+                    if let channel = channel {
+                        self.generalChannel = channel
+                        channel.join(completion: { result in
+                            print("Channel joined with result \(result)")
+                            
+                        })
+                    } else {
+                        // Create the general channel (for public use) if it hasn't been created yet
+                        channelsList.createChannel(options: [TCHChannelOptionFriendlyName: "General Chat Channel", TCHChannelOptionType: TCHChannelType.public.rawValue], completion: { (result, channel) -> Void in
+                                if result.isSuccessful() {
+                                    self.generalChannel = channel
+                                    self.generalChannel?.join(completion: { result in
+                                        self.generalChannel?.setUniqueName(defaultChannel, completion: { result in
+                                            print("channel unique name set")
+                                        })
+                                    })
+                                }
+                        })
+                    }
+                })
+            }
+        }
+    }
+    
+    // Called whenever a channel we've joined receives a new message
+    func chatClient(_ client: TwilioChatClient, channel: TCHChannel,
+                    messageAdded message: TCHMessage) {
+        self.messages.append(message)
+
+        DispatchQueue.main.async() {
+            if let delegate = self.chatManagerDelegate {
+                delegate.reloadMessages()
+                if self.messages.count > 0 {
+                    delegate.scrollToBottomMessage()
+                }
+            }
+        }
+    }
+    
+    func getMessages() -> [TCHMessage] {
+        return messages
+    }
+    
+    func sendMessage(_ messageText:String, completion: @escaping (TCHResult, TCHMessage?) -> Void) {
+        if let messages = self.generalChannel?.messages {
+            let messageOptions = TCHMessageOptions().withBody(messageText)
+            messages.sendMessage(with: messageOptions, completion: { (result, message) in
+                completion(result, message)
+            })
+        }
+    }
+    
+    
+    func login(_ identity:String, completion:@escaping (TCHResult)->Void) {
+        // Fetch Access Token from the server and initialize Chat Client - this assumes you are
+        // calling a Twilio function, as described in the Quickstart docs
+        let deviceId = UIDevice.current.identifierForVendor!.uuidString
+        let urlString = "\(tokenURL)?identity=\(identity)&device=\(deviceId)"
```

Comment:
Why does it need a device ID here? We usually do not use that.

Reply:
That was in place for the endpoint_id from IP Messaging - but that looks to have been deprecated in 0.71. Can you confirm that we only need to provide an identity for an access token? If so, we can also modify the Chat Access Token Function Template.